dashboards/foxtrot_service_dashboard.json

```diff
@@ -120,7 +120,208 @@
         "type": "timeseries"
       },
       {
-        "datasource": "Prometheus",
+        "datasource": {
+          "type": "prometheus",
+          "uid": "PBFA97CFB590B2093"
```

Comment:
this will probably not work. should be without uid

"datasource": "Prometheus",

Reply:
this applies to all fields in json file

Reply:
Thank you!